### PR TITLE
feat: add Work and About sections

### PR DESCRIPTION
**Pull request by Mukumbuta**
feat: add work section and a list of projects each with a corresponding button to view project details; add about section, a brief description about me and a corresponding button to view my CV; activate the work and about section buttons; fix linter errors in style.css file.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -52,5 +58,126 @@ <h1>Hey there. I'm Emmanuel</h1>
...
+            has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+          </p>
+        </div>
+        <ul class="skills">
+          <li class="css">css</li>
+          <li class="html">html</li>
+          <li class="boot">Bootstrap</li>
+          <li class="ruby">Ruby</li>
+        </ul>
+          <button type="button" class="btn">See Project</button>
```
Comment (tufoinnkuo10) [inline index.html:79]: - [ ] kindly add a link place holder `(a tag)` so it will make it easier to navigate

File: index.html [hunk 2]
```diff
@@ -52,5 +58,126 @@ <h1>Hey there. I'm Emmanuel</h1>
...
         </ul>
     </section>
-</body>
-</html>
\ No newline at end of file
+
+    <section class="myprojects">
+      <h2 class="project-title">My Recent Works</h2>
+      <article class="myprojects project">
+        <div class="description">
```
Comment (tufoinnkuo10) [inline index.html:65]: - [ ] please use the same font color as that of the figma version so it looks clear. also, your headings look italicized. please fix that.
- [ ] you are missing the first story image. kindly fix that
your version
Screenshot (201)_LI
figma version
Screenshot (200)_LI

File: index.html [hunk 3]
```diff
@@ -52,5 +58,126 @@ <h1>Hey there. I'm Emmanuel</h1>
...
+          <button type="button" class="btn4">See Project</button>
+      </article>
+
+      <section class="myprojects about">
+        <h2>About me</h2>
+        <p class="intro">
+          Hello I'm a software developer! 
+          I can help you build a product, feature or website Look through some of my work and experience! 
+          If you like what you see and have a project you need coded, don't hestiate to contact me.
+        </p>
```
Comment (tufoinnkuo10) [inline index.html:157]: - [ ] kindly center the content of this section.

File: index.html [hunk 4]
```diff
@@ -52,5 +58,126 @@ <h1>Hey there. I'm Emmanuel</h1>
...
+        <button type="button" class="mycv-btn">Get My Resume</button>
+        <section class="mytechs">
```
Comment (tufoinnkuo10) [inline index.html:160]: - [ ] please add this line to differentiate the about me and your skills sections. 
Screenshot (202)_LI

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @Mukumbuta 

There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight
✅ You used the correct Github flow
✅ Descriptive pull request message
✅ linter error checks validated
✅ mobile version is well followed

## Required Changes ♻️♻️
Check the comments under the review

tufoin

Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: index.html (hunk 1 above)
Comment (Mukumbuta) [reply index.html:79]: Hi @tufoinnkuo10 
Thank you for the review. I have a question, though. 
Did you mean I add the (a tag) on all the "See Project" butons?

**Review by cynthiainga — APPROVED** [review]
## STATUS APPROVED :heavy_check_mark: 
Hi @Mukumbuta, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- [ ] No linters errors :heavy_check_mark: 
- [ ] Good commit messages :heavy_check_mark: 
- [ ] Good implementation of all changes required :heavy_check_mark: 
- [ ] Descrptive PR :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**